source/PlayerInfo.cpp

```diff
@@ -577,6 +577,11 @@ void PlayerInfo::IncrementDate()
 	string message = accounts.Step(assets, Salaries());
 	if(!message.empty())
 		Messages::Add(message);
+	
+	// Advance the reload counters on all the player's weapons.
+	int advanceFrames = 24 * 60 * 60 * 60;	
```

Comment:
Update coming tonight. I'm going to see what happens if I do a rebase and squash on an open PR, particularly what happens to the review history. If I mess it up, my apologies, but lessons learnt!

In the update I'm changing this bit so that it's defined in namespace, and is a `static constexpr int FRAMES_PER_DAY`. More in keeping with the code's style, but since this is breaking out for clarity (24 * 60 * 60 * 60 instead of 5184000) it seems cleaner to push that calculation onto compile time with constexpr, instead of making the game calculate it at runtime.

Ideally I'd one day love to see it be `24 * 60 * 60 * FRAMES_PER_SEC`, as defining the frames per second as a constant would make it easier to understand lines like [this ](https://github.com/Elyssaen/endless-sky/blob/feature/IncrementDateReload/source/OutfitInfoDisplay.cpp#L243) one.

(Although in that example of 6000, while `FRAMES_PER_SEC * 100` would help, it's probably easier to tell that the 6000 includes a 60 frames per second than to tell the 100 is the infinite convergence of a geometric series whose constant ratio is [0.99](https://github.com/Elyssaen/endless-sky/blob/feature/IncrementDateReload/source/Ship.cpp#L734). Michael, the rest of us can't work out at a glance that that's what you're doing, we're not that smart!)